dist/pie-chart.js

```diff
@@ -36,6 +40,26 @@
           $utils
             .updatePaths(svg, data, dim, options)
             .updateLegend(svg, data, dim, options);
+
+          currentData = [];
+
+          angular.copy(data, currentData);
+          currentData[0].finalVal = currentData[0].value;
+          currentData[0].value = 0;
+
+          if (interval) {
+            clearInterval(interval);
+          }
+
+          var interval = setInterval(function() {
+            if (currentData[0].finalVal === currentData[0].value) {
+              clearInterval(interval);
+              return
+            }
+            currentData[0].value++;
+            $utils.updateLegend(svg, currentData, dim, options);
+
+          }, (options.duration / gaugeScale));
```

Comment:
What if someone forgets to set duration? Perhaps a sane default of a second would be nice?